.github/workflows/release.yml

```
@@ -206,13 +224,27 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: google-github-actions/auth@v0
+        continue-on-error: true
+        with:
+          credentials_json: ${{ secrets.gcs_bazel_cache }}
       - name: Build manylinux2010 wheels
         run: |
+          if [[ -n $GOOGLE_APPLICATION_CREDENTIALS ]]; then
+            echo -e 'build --remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-release-manylinux-python${{ matrix.python-version }}' >> .bazelrc.user
+            echo -e 'build --google_default_credentials' >> .bazelrc.user
+          fi
+
+          cp $GOOGLE_APPLICATION_CREDENTIALS gcloud-credentials.json
+
           docker run -e LCE_RELEASE_VERSION=${{ github.event.inputs.version }} \
+            -e GOOGLE_APPLICATION_CREDENTIALS=/compute-engine/gcloud-credentials.json \
             -v ${PWD}:/compute-engine -w /compute-engine \
             tensorflow/build:2.8-python${{ matrix.python-version }} \
             .github/tools/release_linux.sh
 
+          rm gcloud-credentials.json
```

How about directly mounting the secret into the docker container?
```suggestion
          docker run -e LCE_RELEASE_VERSION=${{ github.event.inputs.version }} \
            -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/gcloud-credentials.json \
            -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/gcloud-credentials.json:ro \
            -v ${PWD}:/compute-engine -w /compute-engine \
            tensorflow/build:2.8-python${{ matrix.python-version }} \
            .github/tools/release_linux.sh
```

---

That is much nicer indeed, thanks.